addons/main.go

```diff
@@ -147,6 +152,17 @@ func main() {
 		os.Exit(1)
 	}
 
+	tanzuClusterBootstrapReconciler := controllers.NewtanzuClusterBootstrapReconciler(mgr.GetClient(),
+		ctrl.Log.WithName("controllers").WithName("TanzuClusterBootstrap"),
+		mgr.GetScheme(),
+		cniSelectionClusterVarName,
```

Comment:
Can we create a config struct like `addonconfig.Config`? Some more configs will be added.

Reply:
Sure, done